package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Recognizes certain classes of webpack errors and cleans, aggregates and prioritizes them to provide a better Developer Experience",
   "main": "index.js",
   "scripts": {
-    "test": "jest"
+    "test": "eslint --ignore-pattern test/* & jest" 
```

Comment:
`&&` might be best ? else if eslint fails, the test won't fail because of return status ?